packages/grid/src/GridTheme.ts

```diff
@@ -221,6 +225,11 @@ const defaultTheme: GridTheme = Object.freeze({
   // Divider colors between the floating parts and the grid
   floatingDividerOuterColor: '#000000',
   floatingDividerInnerColor: '#cccccc',
+
+  // Databar
+  zeroLineColor: '#6c757d',
+  positiveColor: '#9cdb6c',
+  negativeColor: '#f25a80',
```

Comment:
Where did you get these colours from? The specific colours we use should be defined in `IrisGridTheme`, and I _think_ should be the same as `positive-number-color` and `negative-number-color` (which map to `$red` and `$green`).
For default GridTheme, we should just be using some basic colours (since we're not dependent on the Deephaven theme), something like:
```suggestion
  // Databar
  zeroLineColor: '#888888',
  positiveColor: '#00ffff',
  negativeColor: '#ff0000',
```